.github/renovate.json

```diff
@@ -0,0 +1,20 @@
+{
+    "extends": ["config:base", ":pinOnlyDevDependencies"],
+    "minimumReleaseAge": "7 days",
+    "automerge": false,
+    "automergeType": "pr",
+    "schedule": ["before 5am on Monday"],
+    "packageRules": [
+      {
+        "matchUpdateTypes": ["minor", "patch"],
+        "automerge": false
```

Comment:
I like to automerge minor and patch versions since it’s the default behavior for npm caret versions.

Reply:
Solved in e415ba79f1e9a8e5489b5b0f7f11ffc572a3e37f

Reply:
https://github.com/yeoman/.github/pull/8#discussion_r1923794520 - I also like this, but IMO we should wait given how old a lot of packages are.